adapters/vm2core/vm2core.go

```diff
@@ -87,3 +87,65 @@ func AdaptStateDiff(fromStateDiff *vm.StateDiff) core.StateDiff {
 	}
 	return toStateDiff
 }
+
+func Receipt(fee *felt.Felt, txn core.Transaction,
+	trace *vm.TransactionTrace, txnReceipt *vm.TransactionReceipt,
+) core.TransactionReceipt {
+	// Determine fee unit based on transaction version
+	feeUnit := core.WEI
+	if txn.TxVersion().Is(3) {
+		feeUnit = core.STRK
+	}
```

Comment:
Would it be relevant to add a transaction v3? Not to test this code spceifically but test everything with Transaction V3 🤔, or is only this function that's missing tests with v3 transactions?

If that's the case then we can ignore this

Reply:
Not important, let's ignore it for now